rest/route/project.go

```diff
@@ -1007,13 +1007,12 @@ func (h *getProjectTasksHandler) Parse(ctx context.Context, r *http.Request) err
 			return errors.Wrap(err, "error parsing request body")
 		}
 	}
-
+	if h.opts.NumVersions < 0 {
```

Comment:
Can this be renamed to limit? I think that may be more in line with our existing routes.